src/AppInfoViewSaver.vala

```diff
@@ -93,7 +93,7 @@ public class AppEditor.AppInfoViewSaver : Object {
 
         key.set_string (KeyFileDesktop.GROUP, KeyFileDesktop.KEY_TYPE, KeyFileDesktop.TYPE_APPLICATION);
         key.set_locale_string (KeyFileDesktop.GROUP, KeyFileDesktop.KEY_NAME, lang, name);
-        key.set_locale_string (KeyFileDesktop.GROUP, KeyFileDesktop.KEY_FULLNAME, lang, name);
+//        key.set_locale_string (KeyFileDesktop.GROUP, KeyFileDesktop.KEY_FULLNAME, lang, name);
```

Comment:
Could you explain the reason behind this change? Does removing this line fix #29? If you are willing to remove it, don't comment it and just remove it althogether.

Reply:
I'm using Manjaro Linux (Arch Linux for humans 😄 ) with Gnome 3. Somehow, this error prevents compiling on my system, so I commented out this line. Everything works with this little change.